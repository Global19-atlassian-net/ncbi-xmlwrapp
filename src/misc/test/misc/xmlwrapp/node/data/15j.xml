<?xml version="1.0"?>
<root xmlns:a="one" xmlns:b="two">
    <b:child xmlns:a="one" xmlns:b="two"/>
</root>
