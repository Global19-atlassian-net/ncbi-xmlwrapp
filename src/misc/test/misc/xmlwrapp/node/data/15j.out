<?xml version="1.0"?>
<root xmlns:a="one" xmlns:b="two">
  <b:child/>
</root>

